ObedZac/iREPORTER
# Ft create new redflag 162218893

# 162218893 create new red flag end point

### *What does this PR do*?

This PR creates the end point for the API to get a create a new red flag
the test for creating a new red flag should pass

### *Description of Task to be completed*?

The app should create a new red flag

### *How should this be manually tested*?

create a virtual environment 
activate the virtual environment
install requirements using pip install -r requirements.txt
run pytest and the test for creating a new red flag should pass
or run the app via python run.py / flask run and test on postman

### *What are the relevant pivotal tracker stories*?

162218893
